src/libraries/Microsoft.PowerFx.Core/Localization/ErrorResourceKey.cs

```diff
@@ -11,7 +12,7 @@ namespace Microsoft.PowerFx.Core.Localization
     /// Existing keys for error messages are split between here (for general document errors) and Strings.cs (for Texl errors).
     /// </summary>
     [ThreadSafeImmutable]
-    public struct ErrorResourceKey
+    public struct ErrorResourceKey : IEquatable<ErrorResourceKey>
```

Comment:
>IEquatable<ErrorResourceKey> [](http://example.com/codeflow?start=37&length=28)

Did a rule flag to add this?

Reply:
Yes
[https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1815](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1815)